ts/src/binance.ts

```diff
@@ -9091,4 +9093,215 @@ export default class binance extends Exchange {
             'info': interest,
         };
     }
+
+    async fetchMyLiquidations (symbol: string, since: Int = undefined, limit: Int = undefined, params = {}) {
+        /**
+         * @method
+         * @name binance#fetchMyLiquidations
+         * @description retrieves the users liquidated positions
+         * @see https://binance-docs.github.io/apidocs/spot/en/#get-force-liquidation-record-user_data
+         * @see https://binance-docs.github.io/apidocs/futures/en/#user-39-s-force-orders-user_data
+         * @see https://binance-docs.github.io/apidocs/delivery/en/#user-39-s-force-orders-user_data
+         * @param {string} symbol unified CCXT market symbol
+         * @param {int} [since] the earliest time in ms to fetch liquidations for
+         * @param {int} [limit] the maximum number of liquidation structures to retrieve
+         * @param {object} [params] exchange specific parameters for the binance api endpoint
+         * @param {int} [params.until] timestamp in ms of the latest liquidation
+         * @returns {object} an array of [liquidation structures]{@link https://github.com/ccxt/ccxt/wiki/Manual#liquidation-structure}
+         */
+        await this.loadMarkets ();
```

Comment:
also, we can add pagination here, probably the incremental works better in this case

Reply:
Sounds good, I added it for spot only on binance because it's the only endpoint that has a pagination related param

Reply:
@carlosmiei Which pagination method should we use with deribit and its `continuation` param?
https://docs.deribit.com/#public-get_last_settlements_by_currency
https://docs.deribit.com/#private-get_settlement_history_by_instrument

Reply:
@Dan-krm cursor like we do at bybit, but I will add it nw